example.env

```diff
@@ -1,6 +1,11 @@
 PERSIST_DIRECTORY=db
-MODEL_TYPE=GPT4All
-MODEL_PATH=models/ggml-gpt4all-j-v1.3-groovy.bin
-EMBEDDINGS_MODEL_NAME=all-MiniLM-L6-v2
-MODEL_N_CTX=1000
-TARGET_SOURCE_CHUNKS=4
+MODEL_TYPE=LlamaCpp
+MODEL_PATH=/path/for/model
+#best english embeddings model
+#best italian efederici/sentence-it5-base
+EMBEDDINGS_MODEL_NAME=all-mpnet-base-v2
```

Comment:
I think that this one uses `768` dimensionalities and might not work with most models like `ggml-vic13b-q5_1.bin` or `koala-7B.ggmlv3.q8_0.bin`.

Reply:
I was also concerned about this. But it seems to work well in my tests. I am using it to take formulas from my physics book while studying hahaha.
Sometimes he gets it wrong, but I'm not sure it depends on this

Reply:
Hello DanieluSG could you make sure about it whether it will fail or not? With this many users don't you agree that it wouldn't be nice to risk it?